MARKETProtocol/dApp
# Added toggle button to Wallet Section

##### Description
1. Added toggle button for Wallet Section
2. Lifting up state from child ( HeaderMenu ) to parent ( Wallet ) for getting collateral and trading balances
3. Added styles for buttons.
4. Added test case for Wallet component and move one test case from HeaderMenu to Wallet for getting balances on contract change

##### Checklist
- [x] Tests coverage hasn't decreased

##### Affected core subsystem(s)
1. Wallet Section UI has new toggle button which can collapsed and expanded.
2. Test case for HeaderMenu is moved to Wallet component since getting balances is lifted up to 
Wallet section

##### Screenshots (Optional)
**Before Change**
![befor_change](https://user-images.githubusercontent.com/42678276/44625267-f0478000-a922-11e8-984c-2d516c8d5e72.png)

**After Change**
![expanded](https://user-images.githubusercontent.com/42678276/44625270-035a5000-a923-11e8-871a-fca9934771cc.png)
![collapsed](https://user-images.githubusercontent.com/42678276/44625274-11a86c00-a923-11e8-8fdb-d69befee6361.png)

##### Testing
**Test Path**
1. Navigate to Exchange Section
2. Wait till all balances are fetched for Wallet Section
3. Toggle Hide / Show Wallet button to see the new UI changes which collapses/expands on toggle.

##### Refers/Fixes
Fixes: #314
